docker-cloud/cloud-swarm/index.md

```diff
@@ -2,164 +2,27 @@
 description: Work with swarms in Docker Cloud
 keywords: swarm mode, swarms, orchestration Cloud, fleet management
 title: Swarms in Docker Cloud
+notoc: true
 ---
 
-<br>
+Docker Cloud now allows you to connect to clusters of Docker Engines running in v1.13 [swarm mode](/engine/swarm/).
 
-<b>Note</b>: All Swarm management features in Docker Cloud are free while in Beta.
-{: style="text-align:center" }
+* [Using swarm mode with Docker Cloud](using-swarm-mode.md)
 
---------
-Docker Cloud now allows you to connect to clusters of Docker Engines running in v1.13 [Swarm Mode](/engine/swarm/).
+* [Swarm mode and organizations](using-swarm-mode.md#swarm-mode-and-organizations)
 
-## Enable Swarm Mode in Docker Cloud
+* [Register existing swarms](register-swarms.md)
 
-By default, Docker Cloud allows you to manage [node clusters](/docker-cloud/infrastructure/), but you can opt in to use the Beta
-Swarm Mode features. Swarm management replaces the node cluster management
-features when the Swarm Mode interface is enabled.
+* [Swarm statuses in Docker Cloud](register-swarms.md#swarm-statuses-in-docker-cloud)
 
-Click the **Swarm Mode** toggle to enable the swarm mode interface.
+* [Connect to a swarm through Docker Cloud](connect-to-swarm.md)
 
-![the Swarm Mode toggle](images/swarm-toggle.png)
+* [Unregister a swarm from Docker Cloud](register-swarms.md#unregister-a-swarm-from-docker-cloud)
```

Comment:
Remove the following items from the navigation list:
* "Swarm statuses"
* "Unregister"
* "Reconnect"